cumulus/client/cirrus-executor/src/aux_schema.rs

```diff
@@ -47,18 +59,28 @@ pub(super) fn write_execution_receipt<Backend: AuxStore, Block: BlockT>(
 		new_first_saved_receipt = block_number.saturating_sub((PRUNING_DEPTH - 1).saturated_into());
```

Comment:
In these cases I would honestly prefer `.expect()` since we don't expect these numbers to ever be beyond `u32::MAX`.

Reply:
Feel free to refactor it directly as I don't see another style that is satisfying to me using `expect()` :P.